cmd/antrea-controller/controller.go

```diff
@@ -106,11 +111,51 @@ func run(o *Options) error {
 	// Set up the antctl server for in-pod access.
 	antctlServer.Start(nil, controllerMonitor, stopCh)
 
+	if o.config.EnablePrometheusMetrics {
+		go createPrometheusMetricsListener(o.config.PrometheusHost,
+			strconv.Itoa(o.config.PrometheusPort),
+			o.config.EnablePrometheusGoMetrics,
+			o.config.EnablePrometheusGoMetrics)
+	}
+
 	<-stopCh
 	klog.Info("Stopping Antrea controller")
 	return nil
 }
 
+func createPrometheusMetricsListener(prometheusHost string,
+	prometheusPort string,
+	enablePrometheusGoMetrics bool,
+	enablePrometheusProcessMetrics bool) {
+	hostname, err := os.Hostname()
+
+	klog.Infof("Initializing prometheus host %s port %s", prometheusHost, prometheusPort)
+	gaugeHost := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "antrea_controller_host",
```

Comment:
some of these can be constants?

Reply:
IDK if that's necessary. Using constant strings here is a common practice (e.g k8s itself, calico etc).
These values won't be used anywhere except in metric definitions so consts won't be reused anywhere - hence it won't enhance readability.